docker-compose.yml

```diff
@@ -16,6 +16,24 @@ services:
         - image-registry.powerapp.cloud/playbook/playbook:main
     volumes:
       - .:/home/app/src
-      - bundle:/usr/local/bundle
+      - bundle:/usr/local/rvm/gems
```

Comment:
Fixes the docker-compose `web` service's bundler cache. The `Dockerfile` explicitly sets the bundler cache to `/usr/local/rvm/gems`

https://github.com/powerhome/playbook/blob/6fcfb7e103ea8029804c35cee7c14a3d92505e53/Dockerfile#L17

This means the `web` service is installing gems onto the container's ephemeral filesystem; when the `web` service's container stops the installed gems get destroyed. 

This fixes the bundle cache path so gems are installed on the bundle volume and retained between container restarts:

```
$ git checkout master \
  && docker compose run --rm -it web bin/bundle check
...
Install missing gems with `bundle install`

$ git checkout 20241007--playbook-website--add-rdbms \
  &&  docker compose run --rm -it web bin/bundle check
...
The Gemfile's dependencies are satisfied
```